### ISSUE TITLE
no default label translation for RadioButtonGroupInput 

### ISSUE THREAD

**Issue by stefsava** [marmelab/admin-on-rest#454]
In RadioButtonGroupInput it seems that the default label translation is not implemented.

**Comment by fzaninotto** [comment]
related to #455

**Comment by stefsava** [comment]
The issue is unrelated to #455, label translation is missing, not choice texts.

Please notice different label for gender:

- "Gender" (**untranslated**) for **RadioButtonGroupInput**
- "Sesso" (translated) for SelectInput
- "Sesso" (translated) for AutocompleteInput

![schermata 2017-03-20 alle 14 51 27](https://cloud.githubusercontent.com/assets/19606/24102589/c6d3e4d6-0d7c-11e7-8b94-5ac9ea07e678.png)

```
import React from 'react';
import {
  Edit,
  SimpleForm,
  DisabledInput,
  TextInput,
  SelectInput,
  RadioButtonGroupInput,
  AutocompleteInput,
} from 'admin-on-rest/lib/mui';

const choices = [
  { id: 'M', name: 'Male' },
  { id: 'F', name: 'Female' },
];

export default props => (
  <Edit {...props}>
    <SimpleForm>
      <DisabledInput source="id" />
      <TextInput source="name" />
      <TextInput source="email" />
      <SelectInput source="gender" choices={choices} />
      <RadioButtonGroupInput source="gender" choices={choices} />
      <AutocompleteInput source="gender" choices={choices} />
    </SimpleForm>
  </Edit>
);
```

```
export default {
  it: {
    resources: {
      core: {
        users: {
          name: 'Utente |||| Utenti',
          list: { title: 'Lista utenti' },
          delete: { title: 'Elimina utente %{name}' },
          fields: {
            name: 'Nome',
            gender: 'Sesso',
          },
        },
      },
    },
  },
};
```

**Comment by fzaninotto** [comment]
Fixed by #501

**Comment by lucasfevi** [comment]
@fzaninotto so I have admin-on-rest@1.2.0 and the RadioButtonGroupInput label isn't being translated... what should I do?

**Comment by fzaninotto** [comment]
It was fixes in 1.2.1.
